### PR TITLE
Update 2D UNet for label-free prediction of mCherry-H2B

### PR DESCRIPTION
**Pull request by bioimageiobot**
This is an automatic PR created by the @bioimageiobot regarding changes to the resource item 10.5281/zenodo.8064806.
The following version(s) will be added:
  - [8064807 (version 1)](https://www.doi.org/10.5281/zenodo.8064807)

Please review the changes and make sure the new item or version(s) pass the following check list:
- [ ] Passed the BioImage.IO CI tests: static (and dynamic) validations
- [ ] The meta information for the RDF item is complete
  - [ ] The tags are complete and describe the model
  - [ ] Naming is intuitive and descriptive, example:  Multi-Organ Nucleus Segmentation (StarDist 2D)
  - [ ] Authors are provided
  - [ ] Documentation is complete
    * For models, include an overview, describe how the model is trained, what is the training data, how to use the model, how to validate the results and list the references. TODO: Model documentation template.
- [ ] Approved by at least one of the BioImage.IO admin team member.

Maintainers: @Justin-Sonneck

Note: If you updated or re-uploaded another version for the current item on Zenodo, this PR won't be changed automatically. To proceed, you can do the following:
 1. Block this version, but keep looking for future versions: Edit the current resource.yaml and keep the top-level `status` field as `accepted`, but change the `status` under the current version to `blocked`.
 2. Accept this version and keep looking for future versions: Merge this PR for now.
 3. Keep proposed version(s) (and this resource in general if it is new) as pending: Close this PR without merging.

 Then wait for the CI on the main branch to complete. It should detect the new version(s) and create another PR for the new version(s).

Previous PRs of this resource: none

### CODE REVIEW

**Comment by FynnBe** [comment]
The dynamic tests are failing, because the `pytorch_state_dict` weights are missing the `depencencies` field specifying a conda environment.yaml file with monai, something like:
```yaml
# rdf.yaml
...
weights:
  pytorch_state_dict:
    ...
    dependencies: conda:environment.yaml 
```
```yaml
# environment.yaml
channels: [conda-forge]
dependencies:
- monai>=1.2;<2.0
```

**Comment by FynnBe** [comment]
Remembering https://github.com/bioimage-io/core-bioimage-io-python/issues/345 I am surprised to see the pytorch_state_dict test with reduced precision and wonder if there are `torchscript` weights as well.
In general we recommend to provide no only `pytorch_state_dict` weights as they have dependencies (here monai) that might not be available in most consumers.

**Comment by Justin-Sonneck** [comment]
Thanks! I have added the environment.yaml and the torchscript weights and revised the rdf.yaml.

**Comment by FynnBe** [comment]
> Thanks! I added the environment.yaml and torchscript weights to the zenodo bucket and revised the rdf.yaml. But I am not quite sure what exactly I need to do now. Do I need to adjust resource.yaml as indicated in point 1 in the auto-update-10.5281/zenodo.8064806 branch?

no, I will block this version 1. Then a new PR will be generated for version 2...
